packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -144,7 +144,6 @@ describe("<DatePicker /> - Focus element", () => {
 
       const input = screen.getByLabelText("Input label")
       await userEvent.tab()
-      expect(input).toHaveFocus()
```

Comment:
One of my rules says no `expect`s outside of `it`s... this is in a `beforeEach`.
I think I just made a quick call that it might not be super important with it being in that `before`.
If you think it is, we can either disable the rule for that line or move things out of the `before` into the actual test.

Reply:
Ohhhh. I think the idea of this check is to ensure that the input is the one we're adding the keyboard action to, not another element. Given the dialog can be opened by the button within the input as well, I suppose it's to prevent a false positive for the first test?

Reply:
Cool. I might change it so it's part of the test.